runtimes.py

```diff
@@ -1,21 +1,26 @@
 import os
 import hashlib
 
+
 CONDA_DEFAULT_LIST = ["tblib", 
                       "numpy", 
                       "pytest", 
                       "Click", 
                       "numba", 
                       "boto3", 
                       "PyYAML", 
-                      "cython"]
+                      'six',
+                      "cython", 'future']
 
 PIP_DEFAULT_LIST = ['glob2', 'boto', 'certifi']
 PIP_DEFAULT_UPGRADE_LIST = ['cloudpickle', 'enum34']
 
 CONDA_ML_SET = ['scipy', 'pillow', 'cvxopt', 'scikit-learn']
 PIP_ML_SET = ['cvxpy', 'redis']
 
+CONDA_OPT_SET = ['scipy', 'cvxopt', ('mosek', 'mosek')]
+PIP_OPT_SET = ['cvxpy' ]
```

Comment:
fyi cvxpy is already in `PIP_ML_SET` - nvm I see that you are trying to build some new optimization specific runtimes. This is fine !